### PR TITLE
Add `lingui-translation-status` action

### PR DESCRIPTION
**Pull request by igorDolzh**
Linear: https://linear.app/pleo/issue/INT-685/create-a-solution-to-see-in-github-pr-the-completion-of-the-translated

I created action which leaves the pull request comment containing information about missing translations per language

To review: https://github.com/pleo-io/actions/pull/60/commits/3ada9cb63f93248b8a9a84ce4431e4a187e5f7f9

Anticipated result:
image

### CODE REVIEW

**Comment by wfortin** [comment]
Looks good but can you point to the files I actually need to review since actions need all of the node_modules committed it's aweful to review this.

**Comment by igorDolzh** [comment]
> Looks good but can you point to the files I actually need to review since actions need all of the node_modules committed it's aweful to review this.

@wfortin Yes, I committed them separately for convenient review https://github.com/pleo-io/actions/commit/3ada9cb63f93248b8a9a84ce4431e4a187e5f7f9 (This is the commit for actual changes)

**Comment by wfortin** [comment]
> > Looks good but can you point to the files I actually need to review since actions need all of the node_modules committed it's aweful to review this.
> 
> @wfortin Yes, I committed them separately for convenient review [3ada9cb](https://github.com/pleo-io/actions/commit/3ada9cb63f93248b8a9a84ce4431e4a187e5f7f9) (This is the commit for actual changes)

@igorDolzh ah sorry I did not see that! Reviewing now :).

File: lingui-translation-status/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+jobs:
+  lingui-status:
+    name: Lingui status
+    env:
+      CI: "true"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Lingui comment
+        id: lingui_comment
+        uses: igorDolzh/lingui-translation-status@0.1.1
```
Comment (wfortin) [inline lingui-translation-status/README.md:23]: I'm wondering if this should live in the pleo-io repo?

Comment (wfortin) [reply lingui-translation-status/README.md:23]: Or was that like a test action and now should be replaced to @pleo-io/lingui-translation-status@0.1.1 ?

File: lingui-translation-status/src/main.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                messages?.forEach((message) => {
+                    content.push(`| ${message} |`)
+                })
+    
+                return title + content.join('\n')
+            })
+    
+            const comment = messagesToPrint?.join('\n\n')
+
+            await gitHub.issues.createComment({
```
Comment (wfortin) [inline lingui-translation-status/src/main.ts:95]: I'm curious if this is going to be too spammy? Some tool that make comments on PRs (like coveralls for example) create a comment and then update it. Might be something we want to look at.

**Review by wfortin — APPROVED** [review]
Nice work! Very happy to see these coming to life.

**Review by Fjandin — APPROVED** [review]

**Pull request merged**